my_list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] Kindly ensure that `MyList` class has the `#initialize` method that accepts multiple arguments as required for the project. For example: 
```ruby
class MyList
    ...

    def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end

    ...
end
```